core/src/main/kotlin/org/eln2/space/Location.kt

```diff
@@ -232,6 +320,9 @@ data class SurfacePos(override val vec3i: Vec3i, val face: PlanarFace) : Locator
 
 	// Preserve chirality: invert _two_ components, or none.
 	// There's very little thought in the permutation otherwise, however; if those need to be changed, they can be.
+	/**
+	 * Orients a vector based on which plane of a cube this node is on.
```

Comment:
This is a good place _not_ to incorporate that comment, but you might want to include a paraphrasing of some of it :)

E.g., "this implementation preserves chirality, but the exact details are left unspecified."